megatron/model/rotary_pos_embedding.py

```diff
@@ -32,7 +35,9 @@ def forward(self, max_seq_len, offset=0):
         emb = torch.cat((freqs, freqs), dim=-1)
         # emb [seq_length, .., dim]
         from einops import rearrange
-        return rearrange(emb, 'n d -> n 1 1 d')
+        base = rearrange(emb, 'n d -> n 1 1 d')
```

Comment:
will this change the output when use ```--use-rotary-position-embeddings```,  in  llama style model, 
 FYI https://github.com/microsoft/Megatron-DeepSpeed/blob/main/examples_deepspeed/pretrain_llama2_distributed.sh?

Reply:
We have tested both GPT and Llama models, this works well with both.